src/basic/log.c

```diff
@@ -1445,6 +1445,54 @@ static const char *const log_target_table[_LOG_TARGET_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(log_target, LogTarget);
 
+void log_task_chain(pid_t pid, const char *special_action_string) {
+        pid_t ppid;
+        uid_t uid;
+
+        const int log_facility_saved = log_facility;
+        const LogTarget log_target_saved = log_get_target();
+        log_set_facility(LOG_AUTHPRIV);
+        log_set_target(LOG_TARGET_SYSLOG);
+        log_open();
+
+        if(pid_is_valid(pid)) {
+                log_notice("System action %s was requested, task chain is printed below:", special_action_string);
```

Comment:
Log notice is not appropriate. This is debug information, hence log_debug.

Reply:
I will change it to log_debug then.